### PR TITLE
RS + RT/YJ/RULE 22-11

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
I'd appreciate it if you reviewed this PR.

### CODE REVIEW

File: rct229/rules/section22/section22rule11.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                        "$": ["pump_power_per_flow_rate"],
+                    },
+                )
+
+            def get_calc_vals(self, context, data=None):
+                child_loop_b = context.baseline
+                secondary_loop_pump_power_per_flow_rate = child_loop_b[
+                    "pump_power_per_flow_rate"
+                ]
+                req_pump_flow_rate = 13 * ureg("W/gpm")
```
Comment (charliepnnl) [inline rct229/rules/section22/section22rule11.py:100]: It would be better to make this a constant near the top of the file.

**Review by charliepnnl — APPROVED** [review]
Approved with a couple of comments that should be addressed.

File: rct229/rules/section22/section22rule11.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/rules/section22/section22rule11.py:100]: Fixed.

File: rct229/rules/section22/section22rule11.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    """Rule 11 of ASHRAE 90.1-2019 Appendix G Section 22 (Chilled water loop)"""
+
+    def __init__(self):
+        super(Section22Rule11, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section22Rule11.ChillerFluidLoopRule(),
+            index_rmr="baseline",
+            id="22-11",
+            description="For Baseline chilled-water system that does not use purchased chilled water, variable-flow secondary pump shall be modeled as 13W/gpm at design conditions.",
+            list_path="ruleset_model_instances[0].fluid_loops[*]",
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rules/section22/section22rule11.py]: Fixed.

**Review by weilixu — APPROVED** [review]
Nicely done, Thanks!

**Pull request merged**